resteasy-spring-web/src/main/java/org/jboss/resteasy/spring/web/ResponseStatusContainerResponseFilter.java

```diff
@@ -7,14 +7,18 @@
 
 public class ResponseStatusContainerResponseFilter implements ContainerResponseFilter {
 
-    private final int status;
+    private final int defaultStatusCode;
+    private final int newStatusCode;
 
-    public ResponseStatusContainerResponseFilter(final int status) {
-        this.status = status;
+    public ResponseStatusContainerResponseFilter(final int defaultStatusCode, final int newStatusCode) {
```

Comment:
This is a public type so we should keep the original constructor otherwise there is a potential migration issue. I don't know if we should just use 200 as the default or not, but I think we should use something like that.

Reply:
We can do that - I'll update the PR tomorrow

Reply:
Fixed